atlante/filestore/file/file.go

```diff
@@ -0,0 +1,115 @@
+package file
+
+import (
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/gdey/errors"
+	"github.com/go-spatial/maptoolkit/atlante/filestore"
+)
+
+const (
+	// TYPE is the name of the provider
+	TYPE = "file"
+
+	// ConfigKeyBasepath is the base directory where the file will be placed.
```

Comment:
It's not clear what config params required / optional. Could you add a Readme to this package or indicate it in the package somewhere.